alf/examples/sac_lag_cargoal1.gin

```diff
@@ -0,0 +1,64 @@
+include 'sac.gin'
```

Comment:
Let's switch to python config from now on.

Reply:
updated